core/libLocator/vmLocator_windows.go

```diff
@@ -1,3 +1,4 @@
+//go:build windows
```

Comment:
why did you add this line?

Reply:
the new format caused by the IDE, I guess